Root Origin update

DrW3RK opened the pull request:
Reference Issue: https://github.com/w3f/polkadot-wiki/issues/3014

DrW3RK committed:
Root Origin update

DrW3RK committed:
Prettify

salmad3 commented on docs/general/glossary.md:
```diff
@@ -469,14 +473,20 @@ bridges).
 
+## Root Origin
+
+A system level [origin](##Origin) in [Substrate](##Substrate). This is the highest privilege level
```
```suggestion
A system-level [origin](##Origin) in [Substrate](##Substrate). This is the highest privilege level
```

salmad3 reviewed (changes requested):
Need to change italic format back to "*" as it will conflict with the conditional rendering on the page

DrW3RK committed:
Revert "Prettify"

This reverts commit e9479d702a0ed01b4229f9f48e51036046b1b84e.

DrW3RK commented:
_Need to change italic format back to "*" as it will conflict with the conditional rendering on the page_

Do you happen to know how to do it? If so, please do it. I reverted the prettify commit and added your suggestion

DrW3RK committed:
hyphenation

salmad3 committed:
format
